app/models/admin_ability.rb

```diff
@@ -145,6 +145,11 @@ def signed_in_with_organizer_role(user, conf_ids_for_organization_admin = [])
     can :manage, Sponsor, conference_id: conf_ids
     can :manage, SponsorshipLevel, conference_id: conf_ids
     can :manage, Ticket, conference_id: conf_ids
+    can :create, TicketScanning do |ticket_scanning|
+      physical_ticket = ticket_scanning.physical_ticket
```

Comment:
this have to written like that because conference id is not a field of the database in this case, right? :thinking: 
what about writing it like:

``` Ruby
conf_id = ticket_scanning.physical_ticket.ticket_purchase.conference_id
conf_ids.include? conf_id
```
I don't think that having the two extra variable are really needed :thinking: